include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -572,6 +581,10 @@ struct OrtCustomOp {
   // Returns the name of the op
   const char*(ORT_API_CALL* GetName)(_In_ struct OrtCustomOp* op);
 
+  // Returns the type of the execution provider
+  // If the function pointer is null, use CPU execution provider by default
+  OrtExecutionProviderType(ORT_API_CALL* GetExecutionProviderType)(_In_ struct OrtCustomOp* op);
```

Comment:
You may use const char* instead of 'OrtExecutionProviderType' at there?

Reply:
OK. I am glad to help if you think `char *` is better.

Reply:
done